style.css

```diff
@@ -0,0 +1,192 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul li {
+  list-style: none;
+}
+
+body {
+  font-size: 16px;
+  line-height: 1.5;
+  color: #333;
+  overflow: hidden;
+  width: 100%;
+  font-family: 'Poppins', sans-serif;
+}
+
+#logo a {
+  color: #6070ff;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+}
+
+/* End of general css */
+
+/* For screens with 375px and below */
+
+@media (max-width: 375px) {
+  .navbar {
+    display: flex;
+    justify-content: space-between;
+    padding: 5%;
+  }
+
+  .menu-items {
+    display: none;
+  }
+
+  .headline_section {
+    background-image: url('/images/header-shapes-mobile.svg');
+    display: flex;
+    padding: 20% 5%;
+    gap: 12px;
+    flex-direction: column;
+  }
```

Comment:
- [ ] Great job on adding the background image for the headline section!
- [ ]  However, it seems that the image is not visible in the UI.
- [ ]  To align with the suggested Figma design, please ensure that the background image is displayed in the UI. This will greatly enhance the overall appeal of the design.

Additionally, remember that your website should precisely match the design provided in Figma, including elements such as the _font_, _colors_, _images_, _text_, and _margins_. You can refer to [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) for guidance on achieving the desired look.
 Keep up the great work!
Your design 
![your](https://github.com/Emeryherve/setup-and-mobile-first/assets/118761303/c2f0e1ef-c2b9-4888-86f2-066e9b833534)
Figma
![mobile](https://github.com/Emeryherve/setup-and-mobile-first/assets/118761303/b767f662-ad67-446e-a480-83e8a3e7f8c1)